lib/decorators/component-list.js

```diff
@@ -85,13 +86,32 @@ function isDraggable(handle, container) {
     || !handle.hasAttribute(refAttr) && getComponentDepth(handle, container) === 1;
 }
 
-function addDragula(el) {
-  const drag = dragula([el], {
-    deadzone: 50,
-    ignoreInputTextSelection: false, // we're handling this in the `invalid` function below
-    moves(selectedItem, container, handle) {
-      const uri = selectedItem.hasAttribute(refAttr) && selectedItem.getAttribute(refAttr),
-        isPageEditMode = _.get(store, 'state.editMode') === 'page';
+function addSortableJS(el) {
+  if ( el.classList.contains('kiln-internals')) {
+    // no need to create Sortable for kiln-
+    return false;
+  }
+  let dragDelay = 0;
```

Comment:
Is there a way to avoid repeating the code that set the dragDelay?

Reply:
Created a new helpers.js file in decorators moved the dragDelay function into that.